spec/services/state_service_spec.rb

```diff
@@ -139,4 +139,28 @@
       end
     end
   end
+
+  describe "#accessioned?" do
+    subject(:accessioned?) { service.accessioned? }
+
+    context "if the accessioned lifecycle exists" do
+      before do
+        allow(workflow_client).to receive(:lifecycle).with(druid:, milestone_name: "accessioned").and_return("2022-04-20 21:55:25 +0000")
+      end
+
+      it "returns true" do
+        expect(subject).to be true
+      end
+    end
+
+    context "if the accessioned lifecycle does not exist" do
+      before do
+        allow(workflow_client).to receive(:lifecycle).with(druid:, milestone_name: "accessioned").and_return(nil)
+      end
+
+      it "returns false" do
+        expect(subject).to be false
```

Comment:
Likewise above:

```suggestion
        expect(service).not_to be_accessioned
```

Reply:
fine with me either way, but if you look up at the other tests in this file, they all uses this `subject` type syntax

should I change them all to match?

Reply:
If you're inclined, sure, that'd be swell 🙂

Reply:
Service with a smile complete